src/Microsoft.OpenApi/Microsoft.OpenApi.csproj

```diff
@@ -33,8 +33,15 @@
     <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
         <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
     </PropertyGroup>
+    <ItemGroup>
+      <PackageReference Include="JsonSchema.Net" Version="4.1.5" />
+      <PackageReference Include="JsonSchema.Net.OpenApi" Version="1.1.0" />
+      <PackageReference Include="SharpYaml" Version="2.1.0" />
+    </ItemGroup>
     <ItemGroup>
       <PackageReference Include="System.Text.Json" Version="7.0.2" />
+      <PackageReference Include="Yaml2JsonNode" Version="1.1.1" />
+      <PackageReference Include="YamlDotNet" Version="13.1.0" />
```

Comment:
Apart from the fact that these two references cannot be in this project, we can't have a reference to both SharpYaml and YamlDotNet. They do the same thing.  If Yaml2JsonNode has take a dependency on YamlDotNet, then we are going to have to rewrite that code.